src/model/internal-messages.model.ts

```diff
@@ -17,26 +22,36 @@ export interface RegisterDataMnemonic extends RegisterDataBase {
   mnemonic: Mnemonic
 }
 
-export interface RegisterDataPrivateKey extends RegisterDataBase {
-  privateKey: PrivateKey
-}
+export type RegisterData = RegisterDataMnemonic
 
-export type RegisterData = RegisterDataMnemonic | RegisterDataPrivateKey
+export interface ImportAccountData {
+  name: string
+  password: string
+  mnemonic: Mnemonic
+  network: Network
```

Comment:
why network is needed for `ImportAccountData`?

Reply:
Yes, it makes more sense to choose network on login screen instead.